app/views/candidate_interface/gcse/science/grade/awards_edit.html.erb

```diff
@@ -0,0 +1,32 @@
+<% content_for :title, title_with_error_prefix(t('gcse_edit_grade.page_title', subject: @subject), @gcse_grade_form.errors.any?) %>
+<% content_for :before_content, govuk_back_link_to %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= form_with model: @gcse_grade_form, url: candidate_interface_gcse_science_edit_grade_path, method: :patch do |f| %>
+      <%= f.govuk_error_summary %>
+
+      <h1 class="govuk-heading-xl"><%= t('gcse_edit_grade.page_title', subject: @subject) %></h1>
+
+      <%= render CandidateInterface::GcseGradeGuidanceComponent.new(@subject, @qualification_type) %>
+      <div class="govuk-grid-row">
+        <div class="govuk-grid-column-two-thirds">
+          <%= f.govuk_radio_buttons_fieldset :gcse_science, legend: { text: "Select the GCSEs you did and include your grade"} do %>
+            <%= f.govuk_radio_button(:gcse_science, ApplicationQualification::SCIENCE_SINGLE_AWARD, label: { text: "Single award" }, hint: { text: "You have one science grade" }, link_errors: true ) do %>
+              <%= f.govuk_text_field :single_award_grade, label: { text: "Grade"}, hint: { text: "For example, 'C' or '4'" }, width: 2 %>
```

Comment:
These strings should be localised, and also, use smart quotes, i.e.

```suggestion
              <%= f.govuk_text_field :single_award_grade, label: { text: "Grade"}, hint: { text: "For example, ‘C’ or ‘4’" }, width: 2 %>
```

Reply:
👍